documentation/index.html

```diff
@@ -13686,6 +13686,8 @@ <h2 id="style/edge-arrow">Edge arrow <a href="#style/edge-arrow"><span class="fa
 <li><code>circle</code></li>
 <li><code>diamond</code></li>
 <li><code>chevron</code></li>
+<li><code>thick-tee</code></li>
+<li><code>crows-feet</code></li>
```

Comment:
Docs should be edited in the md files.  The html is just generated from a template and would be overwritten.